.github/workflows/linux.yml

```diff
@@ -348,7 +347,7 @@ jobs:
           python -m pip wheel -v --no-deps --wheel-dir ${BUILD_DIR} \
               --config-settings=override=cross.arch="manylinux_2_31_x86_64" \
               --config-settings=override=cmake.options.BUILD_FAST_TOKENIZERS="${{ matrix.build_fast_tokenizers }}" \
-              --find-links ${INSTALL_DIR}/tools \
+              --find-links ${OV_ROOT}/tools \
```

Comment:
Perhaps it makes sense to make find-links a separate OV provider output. This way when we change wheels folder location in OV artifacts we won't need to edit tokenizers pipeline here

Reply:
I suppose it will not work with nightly? because they don't have wheels inside `tools` folder

Ideally, we need to have the same workflow file independently on artifacts `revision`

Reply:
That's also why I'm thinking of adding find-links output to OV provider

Reply:
Added a separate output for indicating a wheel source, since we need it here https://github.com/openvinotoolkit/openvino/pull/26517